src/annotator/test/pdf-sidebar-test.js

```diff
@@ -0,0 +1,238 @@
+import $ from 'jquery';
+
+//import events from '../../shared/bridge-events';
+
+import PdfSidebar from '../pdf-sidebar';
+import { $imports } from '../pdf-sidebar';
+
+describe('PdfSidebar', () => {
+  const sandbox = sinon.createSandbox();
+  let CrossFrame;
+  let fakeCrossFrame;
+
+  let fakePDFViewerApplication;
+  let fakePDFContainer;
+  let fakePDFViewerUpdate;
+  const sidebarConfig = { pluginClasses: {} };
+
+  const createPdfSidebar = config => {
+    if (!config) {
+      config = {};
+    }
+    config = Object.assign({}, sidebarConfig, config);
```

Comment:
tiny nit, it would be nice to see more ES6 destructuring when we can...

```
config = {
  ...sidebarConfig, 
  ...config
}
```

Reply:
FWIW, this was copypasta from the `sidebar-test.js` module, as the setup for testing `PDFSidebar` relies heavily on `sidebar-test` (recall that `PDFSidebar` extends `Sidebar`—there's a lot of state/setup required). But yep, I can change it.